cmd/lambda/accounts/create.go

```diff
@@ -105,7 +105,13 @@ func CreateAccount(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// Add Account to Reset Queue
-	err = Queue.SendMessage(&resetQueueURL, &account.ID)
+	body, err := json.Marshal(&account)
```

Comment:
This plus the next 4-5 lines will go away as we make more changes.  This is repeated here as it is the best way to do it at this point.